src/ScrumBoardItBundle/Services/AbstractApi.php

```diff
@@ -64,6 +68,44 @@ protected function call($url)
         return json_decode($content);
     }
 
+    protected function send($url, $content, $nbArguments)
```

Comment:
$content typed variable to array (to avoid another types as string: do you know what i mean ?)